content/en/real_user_monitoring/reactnative/mobile_vitals.md

```diff
@@ -20,27 +20,35 @@ Real User Monitoring offers Mobile Vitals, a set of metrics that can help comput
 | Application not responding     | When the UI thread of an application is blocked for more than 5 seconds, an `Application Not Responding` (ANR) error triggers. If the application is in the foreground, the system displays a dialog modal to the user, allowing them to force quit the application. <br /><br />   RUM tracks ANR occurrences and captures the entire stack trace that blocks the main thread when it encounters an ANR.                                                                                                                                                                                                                              |
 | Crash-free sessions by version | An [application crash][6] is reported due to an unexpected exit in the application typically caused by an unhandled exception or signal. Crash-free user sessions in your application directly correspond to your end user's experience and overall satisfaction. <br /><br />   RUM tracks complete crash reports and presents trends over time with [Error Tracking][7]. <br /><br />  With crash-free sessions, you can stay up to speed on industry benchmarks and ensure that your application is ranked highly on the Google Play Store.                                                                                                 |
 | CPU ticks per second           | High CPU usage impacts the [battery life][8] on your users' devices.  <br /><br />  RUM tracks CPU ticks per second for each view and the CPU utilization over the course of a session. The recommended range is <40 for good and <60 for moderate. <br /><br />  You can see the top views with the most number of CPU ticks on average over a selected time period under **Mobile Vitals** in your application's Overview page.                                                                                                                                                                                                                                                                                                                                                        |
 | Memory utilization             | High memory usage can lead to [out-of-memory crashes][9], which causes a poor user experience. <br /><br />  RUM tracks the amount of physical memory used by your application in bytes for each view, over the course of a session. The recommended range is <200MB for good and <400MB for moderate. <br /><br />  You can see the top views with the most memory consumption on average over a selected time period under **Mobile Vitals** in your application's Overview page.                                                                                                                                                                                                                                                                                            |
+| UI refresh rate average | |
```

Comment:
@timmld Are these supposed to be filled in? Or would be be better to remove them until they can be filled in later?

Reply:
sorry, that can be removed - good catch!